File: backend/VocabuAI.Api/Dtos/FlashCards/FlashCardLearningStatisticsDto.cs
namespace VocabuAI.Api.Dtos.FlashCards;

// Keep in sync with frontend/src/domain/dtos/flashcards/FlashCardLearningStatisticsDto.ts

public sealed record FlashCardLearningStatisticsDto(
    int CorrectCountTotal,
    int WrongCountTotal,
    DateTimeOffset? LastAnsweredAt
);

File: frontend/src/domain/dtos/flashcards/FlashCardLearningStatisticsDto.ts
// Keep in sync with backend/VocabuAI.Api/Dtos/FlashCards/FlashCardLearningStatisticsDto.cs
export type FlashCardLearningStatisticsDto = {
  correctCountTotal: number;
  wrongCountTotal: number;
  lastAnsweredAt?: string | null;
};
